tests/test_navbar.py

```diff
@@ -89,6 +102,8 @@ def test_logout_link(self, driver, page):
         login(driver)
 
 
+@markers.smoke_test
+@markers.core_functionality
 class TestOSFHomeNavbar(NavbarTestLoggedOutMixin):
```

Comment:
This isn't a dealbreaker, but a nice-to-have would be to consistify the class names.  Currently, classed that inherit from `NavbarTestLoggedOutMixin` are named `Test**Foo**Navbar` and classes that inherit from `NavbarTestLoggedInMixin` are named `Test**Foo**NavbarLoggedIn`.  It would be dope if `Test**Foo**Navbar` classes were renamed to `Test**Foo**NavbarLoggedOut` so we don't have to code dive to remember which logged-in state they are run with.  Not a blocker for merging this, just extra credit if you have time.

Reply:
Changed in latest commit. How many extra credit points do I get? :)

Reply:
You get fourteen extra credit points which can be redeemed for goods and services!